vignettes/EuroCity-vignette.Rmd

```diff
@@ -0,0 +1,266 @@
+
+![Menu to choose quality of life criteria](prices_menu.png)
+
+In the main panel on the right, a bar plot will be shown which illustrates how much the chosen product cost on average in May 2023 in the chosen cities (prices in €):
+
+![Menu to choose quality of life criteria](plot_prices.png)
+
+---
+
+## 2: Included data sets 
+
+The following data sets are included in `EuroCity`: 
+
+* `cities`: overview over the cities included in the package. This is mainly used for internal computations. 
+* `price_categories`: overview over the products which are included in the living expenses comparison tab. This is also mainly used for internal function calls. 
+* `city_prices`: contains the minimum, maximum, and average price of several goods in each European city (in May 2023).
+* `life_satisfaction`: contains the mean self-reported life satisfaction of each European country between 2003 and 2021. 
+* `qualityOL`: Contains the percentage of a city's inhabitants reporting to be either "rather satisfied" or "very satisfied" with a range of quality of life criteria in their city
+
+Below you find a summary of the 3 main data sets in `EuroCity`: 
+
+### The city_prices data set 
+
+This data frame contains prices of various goods in European cities. Prices are reported in Euro and were retrieved in May 2023. Data was taken from Cost of Living & Prices API. Copyright therefore does not fall under the MIT license but belongs to Cost of Living & Prices API.
+
+Call this data set by running `city_prices`
+
+**Format:**
+
+The data frame includes: 
+
+* `country`: The country name 
+* `city`: The city name (in English)
+* `item_name`: The name of the product
+* `category`: The Category which the product belongs to. Values are "Housing", "Groceries", "Alcoholic Beverages", "Leisure time and going out", "Transportation"
+* `min`: lowest price of the product in the city (in €)
+* `avg`: average price of the product in the city (in €). This value is presented in the shiny app comparison. 
+* `max`: highest price of the product in the city (in €)
+
+```{r city_prices_df}
+head(city_prices)
+```
+
+### The life_satisfaction data set 
+
+This data frame contains a subset of the data of the World Happiness Report 2023. Data were collected yearly from 2003 to 2021 and includes country-wide means of self-reported life satisfaction on a scale of 0 to 10.
+
+Call this data set by running `life_satisfaction`
+
+**Format:**
+
+The data frame includes: 
+
+* `Entity`: the Country name
+* `Year`: Year of measurement
+* `Life_satisfaction`: Country mean of self-reported life satisfaction on a scale of 0 to 10 
+* `Code`: Country Code (used for plotting map in shiny)
+
+```{r life_satisfaction_df}
+head(life_satisfaction)
+```
+### The qualityOL data set
+
+Contains the results of the Report on the Quality of life in European Cities, reported in 2019 and published in 2020.
+
+Call this data set by running `qualityOL`
```

Comment:
Should they use this data also outside of the app? Or is it mainly for in the app (because then maybe you don't have to say this in the vignette? But I'm not 100% sure, maybe it could still be handy.)

Reply:
Good point, thanks for the input!